types/context/BlogContext.ts

```diff
@@ -0,0 +1,14 @@
+export interface BlogContextValue {
```

Comment:
Add the Blog post type according to the data present in `posts.json`.

Reference - [dev.to/nickytonline/strongly-typed-json-in-typescript-5gb2](https://dev.to/nickytonline/strongly-typed-json-in-typescript-5gb2)